app/components/hearing-form.js

```diff
@@ -0,0 +1,85 @@
+import Component from '@ember/component';
+import { action, computed } from '@ember/object';
+import moment from 'moment';
+
+export default Component.extend({
```

Comment:
we should use class syntax:

`export default class HearingFormComponent extends Component { }`

Reply:
it was es6 class syntax before when it was a controller — why use object literal syntax in the component?